definitions/webbrowser.yaml

```diff
@@ -273,6 +273,7 @@ sources:
     paths:
       - '%%users.localappdata%%\Mozilla\Firefox\Profiles\*\places.sqlite'
       - '%%users.appdata%%\Mozilla\Firefox\Profiles\*\places.sqlite'
+      - '%%users.appdata%%\Roaming\Mozilla\Firefox\Profiles\*\places.sqlite'
```

Comment:
In that case it looks like the Safari location is wrong since it contains Roaming there.  Do you need me to resubmit the PR without the FF change?